cadence/transactions/bridge/admin/evm/set_registrar.cdc

```diff
@@ -29,5 +30,29 @@ transaction(registryEVMAddressHex: String) {
             value: EVM.Balance(attoflow: 0)
         )
         assert(callResult.status == EVM.Status.successful, message: "Failed to set registrar")
+
+        // Confirm the registrar was set
+        let postRegistrarResult = self.coa.call(
+            to: registryEVMAddress,
+            data: EVM.encodeABIWithSignature("registrar()", []),
+            gasLimit: 15_000_000,
```

Comment:
If we're using the gas limit constant in the contract, should we try to use it in transactions also?

Reply:
Yeah good point